index.html

```diff
@@ -859,13 +858,26 @@ <h3>
           application, it SHOULD treat that manifest as a description of a
           distinct application, even if it is served from the same URL as that
           of another application. When the user agent sees a manifest where
-          |manifest|["id"] is [=url/equal=] with [=URL serializer/exclude
-          fragment|exclude fragment true=] to the [=identity=] of an
-          already-installed application, it SHOULD be used as a signal that
+          |manifest|["id"] is [=url/equal=] (OPTIONAL: with
+          [=URL/equals/exclude fragments=] set to true) to the [=identity=] of
+          an already-installed application, it SHOULD be used as a signal that
           this manifest is a replacement for the already-installed
           application's manifest, and not a distinct application, even if it is
           served from a different URL than the one seen previously.
         </p>
+        <p class="note" title="Exclude fragments for good measure">
+          Since the [=process the id member|processing algorithm=] removes the
+          [=URL/fragment=] from the <code>[=id=]</code> member, it is not
```

Comment:
Should this be? (I can't remember... I thought we had scoped them)

```suggestion
          [=URL/fragment=] from the <code>[=manifest/id=]</code> member, it is not
```

Reply:
Yeah this is correct. Unfortunately the file now needs a `tidy` and I'm not at my PC until next week. Let's agree in principle and I'll fix it up and merge next week at my desk.

Reply:
Done. (Wow I was away from my desk for 2 weeks due to illness :sneezing_face: )